packages/vm/src/vm.ts

```diff
@@ -179,10 +179,11 @@ export class VM {
       // put 1 wei in each of the precompiles in order to make the accounts non-empty and thus not have them deduct `callNewAccount` gas.
       for (const [addressStr] of getActivePrecompiles(this._common)) {
         const address = new Address(hexToBytes(addressStr))
-        const account = await this.eei.getAccount(address)
+        let account = await this.eei.getAccount(address)
         // Only do this if it is not overridden in genesis
         // Note: in the case that custom genesis has storage fields, this is preserved
-        if (account.isEmpty() === true) {
+        if (account === undefined) {
```

Comment:
I think this should be
```suggestion
        if (account === undefined || account.isEmpty()) {
```

Such that we do not change behavior.

Reply:
Hmm, before we had the behavior that `getAccount()` *was* just always returning an empty account also for a non-existing one, so we had no choice.

I would argue that coming from the situation here it would just make no sense that an empty account is already present for a precompile, so removing this (previously dirty-included) `account.isEmpty()` seems rather like a clean-up respectively bug fix to me, pretty much safe to be done.

Let me know if I am missing some scenarios.